setup.sh

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 # Description:
-# bash setup.sh MODE={stable,nightly,head,libtpu-only} LIBTPU_GCS_PATH={gcs_path_to_custom_libtpu}
+# bash setup.sh MODE={stable,nightly,head,libtpu-only} LIBTPU_GCS_PATH={gcs_path_to_custom_libtpu} DEVICE={tpu,gpu}
```

Comment:
our automated testing is rejecting this. can you set DEVICE=tpu if it is unset? If that doesn't work let me know.

Reply:
Ill fix these up